content/docs/2.13/scalers/prometheus.md

```diff
@@ -90,7 +90,22 @@ See the follwowing steps to configure the scaler integration.
 - Prometheus server address should follow the Google's Monitoring API for [Prometheus HTTP API](https://cloud.google.com/stackdriver/docs/managed-prometheus/query#api-prometheus):
   - Example: `https://monitoring.googleapis.com/v1/projects/GOOGLE_PROJECT_ID/location/global/prometheus` - where `GOOGLE_PROJECT_ID` should be replaced by your Google project ID.
 
-To gain a better understanding of creating a Prometheus trigger for Google Managed Prometheus, refer to [this example](#example-google-managed-prometheus).
+To gain a better understanding of creating a Prometheus trigger for Google Managed Prometheus, refer to [this example]
+
+
+**Amazon Managed Service for Prometheus:**
```

Comment:
Let's make these cloud-specific offers more discoverable by using headers:
```suggestion
### Amazon Managed Service for Prometheus
```

Please align other cloud providers as well and alphabetically sort them under a new `## Integrating Cloud offerings` section.

Reply:
Thanks for updating, but it looks like we did not align Azure + we did not add the new `## Integrating Cloud offerings` section for this.

Reply:
Updated the alignment for Azure..Not sure what is meant by `## Integrating Cloud offerings`.. I already added an `#### Example: Amazon Managed Service for Prometheus (AMP)` section there and authentication provider already exists for EKS here: https://keda.sh/docs/2.11/authentication-providers/aws-eks/

Reply:
I fixed this.